lib/plugin.js

```diff
@@ -85,10 +85,18 @@ function mergeOptions(options) {
   options = { locale: '<%= options.defaultLocale %>', ...options }
   <% } %>
 
+  <% if (options.fallbackLocale) { %>
+  options = { fallbackLocale: '<%= options.fallbackLocale %>', ...options }
+  <% } %>
+
   if (options && typeof options.locale === 'string') {
     const locale = options.locale.replace(/[_-]/, '')
     if (locales[locale]) {
       options.locale = locales[locale]
+    } else if (options.fallbackLocale !== undefined && options.fallbackLocale !== null) {
+      console.warn(`[date-fns] locale '${options.locale}' not found, using fallback locale '${options.fallbackLocale}'`)
+      const fallbackLocale = options.fallbackLocale.replace(/[_-]/, '')
+      options.locale = locales[fallbackLocale]
     } else {
       console.warn(`[date-fns] locale '${options.locale}' not found.`)
     }
```

Comment:
We can improve this here by checking if the fallback locale also exists.

```js
const locale = options.locale.replace(/[_-]/, '')
const fallbackLocale = options.fallbackLocale ? options.fallbackLocale.replace(/[_-]/, '') : null

if (locales[locale]) {
  options.locale = locales[locale]
} else if (fallbackLocale && locales[fallbackLocale]) {
  console.warn(`[date-fns] locale '${options.locale}' not found, using fallback locale '${options.fallbackLocale}'`)
  options.locale = locales[fallbackLocale]
} else {
  console.warn(`[date-fns] locale '${options.locale}' not found.`)
}
```

Reply:
You're right, I pushed the change :)